internal/provider/service_resource.go

```diff
@@ -328,6 +344,37 @@ func (r *ServiceResource) Create(ctx context.Context, req resource.CreateRequest
 		}
 		return
 	}
+
+	if !plan.MetricExporterID.IsNull() {
+		err := r.client.AttachMetricExporter(ctx, &tsClient.AttachExporterRequest{
+			ServiceID:  service.ID,
+			ExporterID: plan.MetricExporterID.ValueString(),
```

Comment:
this means it can be attached at creation? I'm not sure that's doable in the console / that its an acceptable thing logic wise ?

Reply:
it attaches after the service has been created. if we don't allow this it'll be very inconvenient to use for automation since a user will have to run two terraform applies